src/base/_themes.scss

```diff
@@ -21,6 +21,7 @@
   --theme-color-text-base: #{color.$text-dark};
   --theme-color-text-code: #{color.$base-fuchsia};
   --theme-color-text-emphasis: var(--theme-color-text-base);
+  --theme-color-text-muted: #{color.$base-gray-dark};
```

Comment:
Can we add a `$text-dark-muted` token?

Reply:
There already is a token with that name, but it is much lighter:

![image](https://user-images.githubusercontent.com/13206945/120402081-2e550800-c2f6-11eb-8577-aefdcd8450f7.png)

Original design:

![image](https://user-images.githubusercontent.com/13206945/120402177-65c3b480-c2f6-11eb-9b37-9db97fd3337e.png)

Reply:
@calebeby Let's go ahead and reference the existing token, if we feel it's too light that should embolden us to update the token rather than add an _additional_ subtle/muted color.